eks-distro-base/Dockerfile.minimal-base-nodejs

```diff
@@ -0,0 +1,108 @@
+# syntax=docker/dockerfile:1.4
+# # Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #      http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+
+# # *NOTE* we have to limit our number of layers heres because in presubmits there
+# # is no overlay fs and we will run out of space quickly
+
+################# BUILDER ######################
+ARG BASE_IMAGE=unused
+ARG BUILDER_IMAGE=unused
+ARG BUILT_BUILDER_IMAGE=unused
+ARG MINIMAL_BASE_BUILDER_IMAGE=unused
+ARG VERSIONED_VARIANT
+ARG COMPILER_VERSION=${VERSIONED_VARIANT}
+ARG IS_COMPILER=false
+
+################# DOWNLOAD ON GLIBC ######################
+# Download in the minimal base builder image to allow sharing result across compiler
+ # and runtime images. We also build the cache into the runtime image
+ # which includes the cache data for this layer and along with COPY --link
+ # this also us to ensure this layer is always reusable across the runtime + compiler
+ # builds without relying on the buildkitd local cache
+FROM ${MINIMAL_BASE_BUILDER_IMAGE} as nodejs-download
```

Comment:
this is the layer that ends up getting cached in the runtime image and shared between it and the compiler images